From: Jiri Slaby <jirislaby@gmail.com>
Date: Fri, 17 Mar 2023 20:28:24 +0100
Subject: kvm: drop deprecated and dropped KVM ioctls
Patch-mainline: submitted, https://github.com/kernelslacker/trinity/pull/48
References: 6.8 fix

They were dropped in upstream in commit a5d3df8ae13f (KVM: remove
deprecated UAPIs).

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 ioctls/kvm.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/ioctls/kvm.c b/ioctls/kvm.c
index 30aeed7a..ad6a3bee 100644
--- a/ioctls/kvm.c
+++ b/ioctls/kvm.c
@@ -26,14 +26,7 @@ static const struct ioctl kvm_ioctls[] = {
 	IOCTL(KVM_IRQ_LINE_STATUS),
 	IOCTL(KVM_REGISTER_COALESCED_MMIO),
 	IOCTL(KVM_UNREGISTER_COALESCED_MMIO),
-	IOCTL(KVM_ASSIGN_PCI_DEVICE),
-	IOCTL(KVM_ASSIGN_IRQ),
-	IOCTL(KVM_ASSIGN_DEV_IRQ),
 	IOCTL(KVM_REINJECT_CONTROL),
-	IOCTL(KVM_DEASSIGN_PCI_DEVICE),
-	IOCTL(KVM_ASSIGN_SET_MSIX_NR),
-	IOCTL(KVM_ASSIGN_SET_MSIX_ENTRY),
-	IOCTL(KVM_DEASSIGN_DEV_IRQ),
 	IOCTL(KVM_IRQFD),
 	IOCTL(KVM_CREATE_PIT2),
 	IOCTL(KVM_SET_BOOT_CPU_ID),
@@ -92,7 +85,6 @@ static const struct ioctl kvm_ioctls[] = {
 	IOCTL(KVM_SET_SREGS),
 	IOCTL(KVM_TRANSLATE),
 	IOCTL(KVM_INTERRUPT),
-	IOCTL(KVM_DEBUG_GUEST),
 	IOCTL(KVM_SET_SIGNAL_MASK),
 	IOCTL(KVM_GET_FPU),
 	IOCTL(KVM_SET_FPU),
-- 
2.44.0

